Program.cs

```diff
@@ -1,10 +1,16 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Fibonacci series
+// Given an input eg:n=10, print fibonacci series for f(10)
+
+int first_number = 0;

```

Comment:
Fix this.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/e161fe42-1f31-44e1-b590-e25028cfdb1d)